src/Microsoft.Framework.PackageManager/Restore/RestoreOperations.cs

```diff
@@ -33,20 +33,6 @@ public RestoreOperations(IReport report)
 
             if (node.Item != null)
             {
-                if (node.Library != null &&
```

Comment:
@lodejard can you remember why we have this? With the new code, any resolved library can't have a snapshot as a version. Only RequestedVersions (IVersionSpec) can be a snapshot.